crates/bevy_pbr/src/meshlet/mod.rs

```diff
@@ -214,13 +213,18 @@ impl Plugin for MeshletPlugin {
                     Node3d::EndMainPass,
                 ),
             )
-            .init_resource::<MeshletMeshManager>()
             .insert_resource(InstanceManager::new())
-            .insert_resource(ResourceManager::new(
-                self.cluster_buffer_slots,
-                &render_device,
-            ))
-            .init_resource::<MeshletPipelines>()
+            .add_systems(
+                RenderStartup,
+                (
+                    check_meshlet_features,
+                    (
+                        (init_resource_manager_system, init_meshlet_pipelines).chain(),
+                        init_meshlet_mesh_manager,
+                    ),
+                )
+                    .chain(),
+            )
```

Comment:
take this with an extremely large grain of salt, just wondering if something like this can help maybe? deliberately not a "suggestion"
```rs
            .add_systems(
                RenderStartup,
                (
                    (check_meshlet_features, init_meshlet_mesh_manager).chain(),
                    (init_resource_manager_system, init_meshlet_pipelines).chain()
                        .after(check_meshlet_features),
                )
            )
```